analyze/edge-light.ts

```diff
@@ -161,3 +187,56 @@ export async function detectBot(
     };
   }
 }
+export async function detectSensitiveInfo<Custom extends string>(
+  context: AnalyzeContext,
+  candidate: string,
+  options?: SensitiveInfoConfig<Custom>,
+): Promise<DetectSensitiveInfoResult<Custom>> {
+  const analyze = await init(context, options?.customDetect);
+  const skipCustomDetect = options?.customDetect === undefined;
+  let entities: core.Entities = {
+    tag: "allow",
+    val: options?.allow?.map(ConvertProtocolEntitiesToAnalyzeEntities) || [],
+  };
+  if (options?.deny) {
+    entities = {
+      tag: "deny",
+      val: options.deny.map(ConvertProtocolEntitiesToAnalyzeEntities),
+    };
+  }
```

Comment:
We probably shouldn't care to try to map here and instead should do any transformations in the local rule itself. I've tried to make Analyze mostly just map to WIT after it does the Wasm initialization.